app/assets/javascripts/map/helpers/guides.js

```diff
@@ -159,7 +146,7 @@ define([
 
       // Layers module
       {
-        selector: "#layersnav-forest-change", 
```

Comment:
We [have it](https://github.com/Vizzuality/gfw/blob/master/.editorconfig) !  So maybe it was an old file without the rules